sdk/resourcemanager/Azure.ResourceManager/api/Azure.ResourceManager.netstandard2.0.cs

```diff
@@ -326,6 +326,52 @@ public sealed partial class ArmPlan : System.IEquatable<Azure.ResourceManager.Mo
         public static bool operator ==(Azure.ResourceManager.Models.ArmPlan left, Azure.ResourceManager.Models.ArmPlan right) { throw null; }
         public static bool operator !=(Azure.ResourceManager.Models.ArmPlan left, Azure.ResourceManager.Models.ArmPlan right) { throw null; }
     }
+    public static partial class ArmResourceManagerModelFactory
```

Comment:
This is unexpected. I will fix this in the generator and we could see it fixed in the next regen preview PR. Please ignore this change now.